### DISCUSSION TITLE
I NEED HELP

### DISCUSSION THREAD

**Discussion by anajo333** [sampotts/plyr#2147] [General]
Hello Dears, 
plyr is perfect tool but i need something
i get audio source and video source i want plyr to combine them togther and play them without any problems
i tryed a lot of solutions like popcorn js code https://tackoverflow.com/questions/6433900/syncing-html5-video-with-audio-playback

but i get a lot of problems with sync sources togther
![image](https://user-images.githubusercontent.com/37766035/111572743-e1ab6780-87b1-11eb-8edb-90f7717c69ea.png)
audio in last source

**Comment by g000m** [comment]
I'm working on something similar- synching one audio track to a video. Did you make any progress @anajo333 ?
